modules/core-updates/8.0.md

```diff
@@ -15,16 +15,25 @@ menuTitle: Changes in 8.0
 
 ## PHP support
 
-PrestaShop 8.0 adds supports for PHP 8.0 and requires at least PHP 7.2.5.
+PrestaShop 8.0 adds supports for PHP 8.0 and PHP 8.1 and requires at least PHP 7.2.5.
 
 ## Notable changes
 
 * Javascript files for migrated pages in the Back Office [now are written in TypeScript](https://github.com/PrestaShop/PrestaShop/pull/23221) (related change: [#24791](https://github.com/PrestaShop/PrestaShop/pull/24791)).
 * Support for [legacy translation dictionaries][legacy-translation] (eg: `en.php`, `fr.php`..., deprecated in 1.7), [has been removed](https://github.com/PrestaShop/PrestaShop/pull/24401) from Core controllers. Only modules can still use this system now.
-* All modules present in the `modules` directory during the installation process will be installed by default. Modules can use `Module::postInstall()` to execute code after the shop installation is done ([Read more](https://github.com/PrestaShop/PrestaShop/pull/24676)).
+* All modules present in the `modules` directory during the web-based installation process will be installed by default [unless the user chooses otherwise](https://github.com/PrestaShop/PrestaShop/pull/27099). Modules can use `Module::postInstall()` to execute code after the shop installation is done ([Read more](https://github.com/PrestaShop/PrestaShop/pull/24676)).
```

Comment:
Why specify the `web-based` here, it's the same thing when you install by CLI so no need to specify this here right? If it's about the possibility to unselect modules then we maybe need to be more accurate like:

`All modules present in the `modules` directory during the installation process will be installed by default (except in the web-based installation where you can unselect some of them https://github.com/PrestaShop/PrestaShop/pull/27099).`